.travis.yml

```diff
@@ -15,47 +15,26 @@ python:
   - "2.7"
 node_js:
   - "0.10"
+env:
+  - RUN_TEST=jshint
+  - RUN_TEST=unittest
+  - RUN_TEST=uitest-phantom-ci
+  - RUN_TEST=uitest-slimer-ci
+  - RUN_TEST=test-langpacks
+  - RUN_TEST=test-package
 before_script:
+  - "export SLIMERJSLAUNCHER=$(which firefox)"
```

Comment:
Good call, Travis uses whatever ESR is so that should be pretty stable. Worth doing both though. I'll see if I can get that working.